src/multidim/setdiff.jl

```diff
@@ -105,3 +105,25 @@ end
 #
 #     return results_list
 # end
+
+import IntervalArithmetic.hull
```

Comment:
You don't need this, since we're still in the `IntervalArithmetic` module. You're just adding another method to the `hull` function.